client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -394,6 +399,10 @@ interface ChatClient {
                 throw IllegalStateException("apiKey is not defined in " + this::class.java.simpleName)
             }
 
+            if (BuildConfig.DEBUG) {
+                Stetho.initializeWithDefaults(appContext)
```

Comment:
Could we keep Stetho installation into another different PR?

Reply:
Yes, removed it from here